### DISCUSSION TITLE
Can I retrieve the cursor value of another sensor in a sensor definition?

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17514] [Q&A]
Hi everyone,

Do someone know if, there’s a way I could retrieve the cursor value of another sensor in a sensor definition?

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1698762127445059?thread_ts=1698762127.445059&cid=C01U954MEER).*

**Comment by marcilj** [comment]
I've browser the `contexte.instance` object in order to find something that will give me the current cursor value for a sensor and I failed completly.

Is there a function that allow me to retrieve that?

I'm trying to start the execution of a job based on the cursor of other sensors.

I was trying to use the Automaterialization Feature instead, but the available condition aren't enough for my usecase.
